### DISCUSSION TITLE
What's the best way of relate the internal User table with relational Database?

### DISCUSSION THREAD

**Discussion by peterhuppertz** [Budibase/budibase#9042] [Help] [answered]
I was searching the discussions and docs before since I guess it's a very common question. So apologies if I missed the answer.
I use MySQL for most of my data to increase performance of complex queries. In some tables I need to relate to data from the internal user table, e.g. providing a list of people to choose from. What is the best way to do this? These are my approaches so far:
1) Create an automation to sync the User table with a table in MySQL and use that table for all relations
2) Add the User table as data provider to screens and save the _id as text field in the MySQL table and filter accordingly to display the data

Maybe I miss another / more clean solution?

**Comment by andz-bb** [comment, marked as answer]
hey @peterhuppertz, I think the second option you mentioned is going to be the best in this case. I set it up on my end with the User _id as a text field on my external table to test it out. It's all working fine with any filtering based on Current User bindings etc. If you run into any issues with that approach let me know and I can try and help you out with it 😊
